src/dvclive/utils.py

```diff
@@ -127,3 +129,19 @@ def inside_notebook() -> bool:
 
         return IPython.__version__ >= "6.0.0"
     return False
+
+
+def get_unmatched_name_from_path(input_path, existing_names):
+    input_path = Path(input_path)
+    name = input_path.stem
```

Comment:
For now we have a regex checking for names https://github.com/iterative/dvc/blob/main/dvc/repo/artifacts.py#L16

we need to make sure dvclive can't produce names that will be invalidated by that regexp. Depending on when you want to merge this, we can revisit this later.

Reply:
DVCLive now requires DVC Python API so I think I would update the P.R. to import the name check function from DVC and use it here. Does that sound good to you?